### DISCUSSION TITLE
Can I use Github Pages for a small static website for my startup business?

### DISCUSSION THREAD

**Discussion by mujtaba-io** [community/community#62762] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello. I am starting a tiny business and github pages is very efficient for it. I only want to ask from legal point of view, that is it legal to use Pages for my business website? I also linked a contact form (embedded from google forms) to the website. Besides, I added some posts to tell viewers what I sell. -- Before pushing changes to my repo, I wanted to confirm. Thank you.

**Comment by GhostOf0days** [comment]
You can, but don't. For startups, use Netlify, Heroku, Vercel, or Railway. Avoid Github Pages. For startups, I mostly suggest Vercel and Railway. For regular projects, Netlify suffices.

**Reply by GhostOf0days** [reply]
They all can deploy from your source code on Github.

**Reply by mujtaba-io** [reply]
Thank you for response. - I choose GitHub because it is easy since I regularly use it. - And I dont know whether my business is going to last or not, since it is kind of an experiment for me. - However, just checked the prohibited use section of GitHub Pages and it says tat we cannot use it for business. But my specific use is to only advertise my business and not to perform real transactions from Pages.

**Reply by mujtaba-io** [reply]
Understood. And again thank you so much.

**Reply by GhostOf0days** [reply]
Again, all of those services, you keep code, etc. on Github. It's just that you deploy with those services rather than Github Pages. They're free and easy to use. Do not use Github Pages for businesses even if it's a small use case. If it's a small use case as you just mentioned, use Netlify. If it's a bigger use case, such as a startup, use Vercel or Railway.
